### ISSUE TITLE
Translate course prerequisite + Nazareth application pages to Arabic and Hebrew

### ISSUE THREAD

**Issue by rebeccaradding** [foundersandcoders/our-site#434]
Will post a Google doc here containing English text as soon as it is finalized!

**Comment by shiryz** [comment]
I believe these are the steps we need:
- [ ] translate to arabic.
- [ ] translate to hebrew.
- [ ] add user story for the translation option.
- [ ] add buttons of He, Ar and En that allows the user to change the language on the page.
- [ ] add the translation page.
- [ ] link the buttons and the pages.

anything else?

I'm happy to do the translation because I don't have a lot of time to work on the website, @esraajb, @Suuuuha are you okay with proofreading and working on the website? :smiley:

**Comment by rebeccaradding** [comment]
@shiryz the text is nearing finalization: https://docs.google.com/document/d/1HEST7ZghnRXyyYfig6MraGPBY_FKI6UWAQ1ZgGCgez8/edit?userstoinvite=hilbertbonamy@gmail.com&ts=59469713&actionButton=1#

You're welcome to use this doc to start your translations, or to create a new one, whatever you prefer!

**Comment by rebeccaradding** [comment]
@shiryz as you can see, site is up! Anything you need from me for the translations?

**Comment by shiryz** [comment]
@rebeccaradding  working on the [arabic translation](https://docs.google.com/document/d/1lf_vigig61q3sb8DZvi7Zdt4aMFH0Gw3kEQ3CAj_WUo/edit?usp=sharing) but this is taking a lot longer than I thought, if someone could get a head start on the hebrew one it would be great :smile:

**Comment by rebeccaradding** [comment]
@shiryz thank you for the update! Help with Hebrew translation, anyone? @HilbertSpitzer?

**Comment by HilbertSpitzer** [comment]
> @shiryz thank you for the update! Help with Hebrew translation, anyone? @HilbertSpitzer?

Did I become the go to guy for these things?
Alright, I'm on it
If I finish it by tomorrow can I get awarded with a black fac shirt? At low price maybe?

**Comment by rebeccaradding** [comment]
@HilbertSpitzer one black FAC shirt coming your way @mattlub @m4v15 please add it to the packing list 🥇

**Comment by shiryz** [comment]
[Arabic translation](https://docs.google.com/document/d/1lf_vigig61q3sb8DZvi7Zdt4aMFH0Gw3kEQ3CAj_WUo/edit?usp=sharing) done!, @Suuuuha, @hibaakel, @HilbertSpitzer  can you read it and fix any typos?

**Comment by HilbertSpitzer** [comment]
> Arabic translation done!, @Suuuuha, @hibaakel, @HilbertSpitzer can you read it and fix any typos?

Maybe later, but for now...
did you leave the "Apply" thing in English on purpose to attract attention? You could have used:
قدم طلبك الآن
اشترك الآن
نفذ الآن
أبدى الآن
تمم الآن
انجز الآن
عاشر الآن
باشر الآن
F&C انضم ل
الآن F&C  رافق 

We could use sth more provocative:
امتحن خبرتك الآن
امتحن انجازاتك الان
Or a Dreams thing:
نفذ مشروعك الان
حقق مشروعك الان
Or a Drama Queen thing:
خاطر الان
ألحق نفسك واشترك  الان

*cough cough*, needless to say, you don't want me to fix you any errors

**Comment by HilbertSpitzer** [comment]
I changed it just a bit

I was thinking that perhaps near the "Apply Now" button we should put a subtext in which we show people quick points about why they should apply in the first place. To do it effectively we would probably need to know what is holding people back from applying in the first place, do they think the course will be too hard? Then why not quickly mention the mentors and C4E or sth. Are they scared they might not know the place? Give them a link that talks about their residence and give them google map images

We should also add a small icon next to the apply thing and an arrow inside the button itself to emphasize that the button really is a link that will take them to a different page entirely and leave the current foundersandcoders page. Perhaps we should use a sort of dropdown thing to describe the "apply" action or gives additional information there as opposed to putting swaths of text next to the button, people want convenience and they would prefer not to read alot, this would satisfy the people who don't like to read as well as the people who would like to read just a bit more

Or am I going overboard with emphasizing the apply thing? I don't know, I kinda have the impression that it's the most important thing there

**Comment by shiryz** [comment]
> we show people quick points about why they should apply

@HilbertSpitzer , I agree we should add that but in the Naz information instead of the apply. we do mention the mentors, c4e, videos and articles in the information pages.

> We should also add a small icon next to the apply thing

I'm keeping the apply consistent with the rest, so we have an apply  section and an apply button on the london page as well, we can change it, whoever does it needs to change all the other buttons.

**Comment by leemordechai** [comment]
@HilbertSpitzer @shiryz @Suuuuha 
bumping this up with a link to the mostly done Hebrew translation - any comments on this would be welcome!
https://docs.google.com/document/d/1OcfYxetPCGLH8_pULpEKevRHOKCvfkwgYn3dU6qx_C8/edit?usp=sharing

**Comment by shiryz** [comment]
@leemordechai Thank you for all this work :heart:! there are a few pages that still need to be translated, I'm happy to do it unless you want to continue working on this

**Comment by leemordechai** [comment]
@shiryz  oops I thought I covered it all! Which ones were you thinking about? I can do them all over the weekend.

**Comment by shiryz** [comment]
@leemordechai the course information pages on Nazareth (3rd page) and the join Nazareth one (page 7).

**Comment by leemordechai** [comment]
@shiryz it should all be updated now, including some of the pages which are currently online but not in the English Google Docs :shipit:
